### ISSUE TITLE
Failed to umarshal type []byte

### ISSUE THREAD

**Issue by socket515** [goccy/go-json#266]
go version: go1.16
example:

```go
import (
        "fmt"
        stdjson "encoding/json"
	"github.com/goccy/go-json"
)

type Request struct {
	Name string `json:"name"`
	PayLoad []byte `json:"pay_load"`
}

func main() {
      bs := []byte(`{"name":"hah","pay_load":"LWItFQcIBQwvDA==\n"}`)
      r := new(Request)
      fmt.Println(json.Unmarshal(bs, r)) // return base64 err
      fmt.Println(stdjson.Unmarshal(bs, r)) // nil
}
```
 std json libary can work, go-json return error

**Comment by socket515** [comment]
Java base64 encode maybe has '\n'

**Comment by goccy** [comment]
Thank you for the reporting ! I fixed this issue with #273
